src-relution/mwFileUpload/mwFileUpload.js

```diff
@@ -267,6 +275,33 @@ angular.module('mwFileUpload', [])
             }.bind(this));
           },
           add: function (e, data) {
+            var hasFileSizeError = false;
+            if (scope.maxFileSizeByte !== undefined) {
+              // check each file if it is too large to upload.
+              angular.forEach(data.files, function (file) {
+                if(file.size >= scope.maxFileSizeByte) {
```

Comment:
If you have a file which is 512kb and the allowed file size is 512kb this check would lead to an error.
it should be  `if(file.size > scope.maxFileSizeByte)`

Reply:
good catch! thanks!

Reply:
just saw that I already fixed that but didn't push 💤